### PR TITLE
Shorthand syntax for empty divs and spans

### PR DESCRIPTION
**Pull request by ssahara**
This PR implement shorter syntax `<WRAP|div|block|wrap|span|inline [classname] />` that allow shorthand for empty divs/spans such as `<WRAP clear/>` or  `<span #id/>`.

This request is replacement of #59 that consist with wired commit history.

### CODE REVIEW

File: syntax/div.php [hunk 1]
```diff
@@ -101,14 +105,16 @@ function render($mode, Doku_Renderer $renderer, $indata) {
                     // include the whole wrap syntax
                     $renderer->startSectionEdit(0, 'plugin_wrap_end');
 
-                    $wrap = plugin_load('helper', 'wrap');
+                case DOKU_LEXER_SPECIAL:
+                    $wrap = $this->loadHelper('wrap');
                     $attr = $wrap->buildAttributes($data, 'plugin_wrap');
 
                     $renderer->doc .= '<div'.$attr.'>';
+                    if ($state == DOKU_LEXER_SPECIAL) $renderer->doc .= '</div>';
```
Comment (selfthinker) [inline syntax/div.php]: That is not necessary, you're already in the `DOKU_LEXER_SPECIAL` case. (The same is true in the `span.php`.) Is that left over from previous tries?

**Comment by selfthinker** [comment]
Yup, this works fine. :+1:

Could you also update [example.txt](https://github.com/selfthinker/dokuwiki_plugin_wrap/blob/master/example.txt)? The new shorthand syntax should be mentioned at the bottom of the "Basic Syntax" section (I guess your `<WRAP clear/>` and `<span #id/>` examples make sense), and the examples for `clear` and `pagebreak` in the "Miscellaneous" section could be updated as well.

File: syntax/div.php (hunk 1 above)
Comment (ssahara) [inline syntax/div.php]: It is necessary because there is no `break` statement before  line 108 `case DOKU_LEXER_SPECIAL`.

**Comment by ssahara** [comment]
OK. please let me see what I can help you on example.txt file.

File: syntax/div.php (hunk 1 above)
Comment (selfthinker) [inline syntax/div.php]: Ah, okay.

**Comment by ssahara** [comment]
Hello @selfthinker 
I have updated the [example.txt](https://github.com/ssahara/dokuwiki_plugin_wrap/blob/shorthand/example.txt) according to your guidance. Also I fixed an inadequate Japanese translation in `ja/lang.php` file, in which my name found as an author!

**Pull request merged**